src/services/travelServices.js

```diff
@@ -52,7 +52,53 @@ export const showManagerPendingAppr = async (manager) => {
   }
 };
 
-export const showUsertravelsStatus = async (userId) => {
+const checkApprovalStatus = async (id, userId) => {
+  try {
+    return await travel_requests.findAll({
+      attributes: ['approval_status'],
+      where: { id, user_id: userId },
+      raw: true
+    });
+  } catch (err) {
+    throw err;
+  }
+};
+
+
+const editOpenRequests = async ({
```

Comment:
You should have created a travel request object in the controller, then pass it as a parameter along with the travel ID. You can check out `oneWayTripService` above as a guide.

Reply:
@chuksjoe .. Another solid point

onewaytrip feature already enforces what it has to get in... here i am not sure what the user will be updating, and the best way to handle it is to cater for all possibilities. Moreover, this service handles all requests including returntrip